src/Admin/MaintainRoles/components/StaffRoleProfile.js

```diff
@@ -14,6 +14,10 @@ class StaffRoleProfile extends Component {
   }
 
   render() {
+    const aDivMiddle = {
+      padding: '10px 15px 5px 15px',
```

Comment:
padding makes the yellow box around the link look very odd since big.
Would be better to use `flex` instead so something like:
```
export const ButtonContainer = styled('div')`
  display: flex;
  align-items: baseline;
`

Reply:
and change the `div` surrounding the `button` and `link` to be `ButtonContainer` instead

Reply:
Thanks!